spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/stream/SkipperStreamDeployer.java

```diff
@@ -220,7 +221,14 @@ public Release deployStream(StreamDeploymentRequest streamDeploymentRequest) {
 		installProperties.setReleaseName(streamName);
 		installProperties.setConfigValues(new ConfigValues());
 		installRequest.setInstallProperties(installProperties);
-		Release release = skipperClient.install(installRequest);
+		Release release = null;
+		try {
+			release = this.skipperClient.install(installRequest);
+		}
+		catch (Exception e) {
+			this.skipperClient.packageDelete(packageName);
```

Comment:
Should we try/catch packageDelete as well? Then it's a choice of logging a warning or choosing which error to wrap in `SkipperException`?